content/ways-to-train.md

```diff
@@ -134,80 +132,94 @@ PGCE programmes combine academic study with a minimum of 24 weeks spent training
 
 You do not need an academic award such as a PGCE to teach in England, but it may be useful if you want to teach in another country.
 
+You can also [get funding](/funding-your-training).
+
 <p class="call-to-action__action">
   <a href="https://www.gov.uk/find-postgraduate-teacher-training-courses">Find a PGCE <span>course</span></a>
 </p>
 
 $school-direct-fee-funded$
 
-School Direct (fee-funded) programmes mean you’ll learn on the job while you work towards your Qualified Teacher Status (QTS). You’ll be based in a school, but you won’t be employed by them. You’ll learn on the job and be taught by the university, college, or school-centred initial teacher training (SCITT) provider that the school partners with.
+You’ll learn on the job while you work towards your Qualified Teacher Status (QTS).
+
+You’ll be based in a school but you will not be employed by them.
 
-Most School Direct programmes lead to a PGCE, but not all do — check with the training provider before applying.
+You’ll also be taught by the university, college, or school-centred initial teacher training (SCITT) provider that the school partners with.
```

Comment:
There are three consecutive paragraphs beginning with the word `You'll` here, and several others below. Perhaps it could be mixed up a bit?